server/athenian/api/controllers/settings.py

```diff
@@ -91,6 +91,48 @@ def _cached_release_settings_sql(account: int, repos: Collection[str]) -> Select
         return select([ReleaseSetting]).where(and_(ReleaseSetting.account_id == account,
                                                    ReleaseSetting.repository.in_(repos)))
 
+    @staticmethod
+    @lru_cache(1024)
+    def _cached_release_settings_for_repos_sql(repos: Collection[str]) -> Select:
+        return select([ReleaseSetting]).where(ReleaseSetting.repository.in_(repos))
```

Comment:
```suggestion
```

We have to persist the account check, the whole design is based on the security model in which each repo is always bound to some account. If you load the repository sets, for example, you should have it as the `owner_id`. This cutting the corners is OK for an experiment but ofc is not OK to include to a production codebase :wink:

Reply:
> This cutting the corners is OK for an experiment but ofc is not OK to include to a production codebase 😉

💯